safekeeper/src/receive_wal.rs

```diff
@@ -397,24 +408,29 @@ async fn read_network_loop<IO: AsyncRead + AsyncWrite + Unpin>(
     loop {
         let started = Instant::now();
         let size = next_msg.size();
-        match msg_tx.send_timeout(next_msg, SLOW_THRESHOLD).await {
-            Ok(()) => {}
-            // Slow send, log a message and keep trying. Log context has timeline ID.
-            Err(SendTimeoutError::Timeout(next_msg)) => {
-                warn!(
-                    "slow WalAcceptor send blocked for {:.3}s",
-                    Instant::now().duration_since(started).as_secs_f64()
-                );
-                if msg_tx.send(next_msg).await.is_err() {
-                    return Ok(()); // WalAcceptor terminated
+
+        tokio::select! {
```

Comment:
nit: this `select` now only has a single branch, and can be removed.

Reply:
+1